src/tweezer.js

```diff
@@ -29,43 +30,44 @@ export default class Tweezer {
   }
 
   on (name, handler) {
-    this.events[name] = this.events[name] || []
-    this.events[name].push(handler)
+    const e = this.events[name]
+    if (e) e.push(handler)
+    else this.events[name] = [handler]
     return this
   }
 
   emit (name, val) {
-    let e = this.events[name]
-    e && e.forEach(handler => handler.call(this, val))
+    const e = this.events[name]
+    if (e) e.forEach(handler => handler.call(this, val))
+    return this
   }
 
   _tick (currentTime) {
     this.isRunning = true
-
-    let lastTick = this.next || this.start
+    const lastTick = this.next === null ? this.start : this.next // this.next can be 0
 
     if (!this.timeStart) this.timeStart = currentTime
     this.timeElapsed = currentTime - this.timeStart
     this.next = Math.round(this.ease(this.timeElapsed, this.start, this.end - this.start, this.duration))
 
     if (this._shouldTick(lastTick)) {
-      this.emit('tick', this.next)
-      this.frame = requestAnimationFrame(this._tick.bind(this))
+      this.emit('tick', this.next).frame = requestAnimationFrame(this._tick)
```

Comment:
I agreee with the PR over all, but doesn't these two simplifications make the code less readable?